### DISCUSSION TITLE
thicker lines for screens with high dpi

### DISCUSSION THREAD

**Discussion by matmay** [OpenHantek/OpenHantek6022#178] [Ideas]
Hello,

for me it is quite difficult to see the scope lines, since i have 14" 3200x1800 screen which makes them very thin. I assume they are just one pixel thick.
Everything else is displayed quite nicely.
Maybe I will also find a better color combination for me. For now I have just made the lines brighter.

Thanks,
Matthias
![image](https://user-images.githubusercontent.com/3354633/113506078-dcd10c80-9542-11eb-850f-ce79069dad5b.png)

**Comment by Ho-Ro** [comment]
Qt doesn't support the OpenGL function to draw thicker lines - only 1 pixel wide lines are supported. This is a known situation :(
